root/components/ReleaseGroupsList.js

```diff
@@ -0,0 +1,263 @@
+    </tr>
+  </thead>
+);
+
+const ReleaseGroupsListEntry = ({
+  $c,
+  checkboxes,
+  index,
+  noGrouping,
+  noRatings,
+  releaseGroup,
+  seriesItemNumbers,
+  showArtists,
+}: ReleaseGroupsListEntryProps) => (
+  <tr className={loopParity(index)} key={releaseGroup.id}>
+    {$c.user_exists && checkboxes ? (
+      <td>
+        <input
+          name={checkboxes}
+          type="checkbox"
+          value={releaseGroup.id}
+        />
+      </td>
+    ) : null}
+    {seriesItemNumbers ? (
+      <td style={{width: '1em'}}>
+        {seriesItemNumbers[releaseGroup.id]}
+      </td>
+    ) : null}
+    <td className="c">
+      {releaseGroup.firstReleaseDate
+        ? new Date(releaseGroup.firstReleaseDate).getFullYear()
+        : '—'}
+    </td>
+    <td>
+      <EntityLink entity={releaseGroup} />
+    </td>
+    {showArtists ? (
+      <td>
+        <ArtistCreditLink
+          artistCredit={releaseGroup.artistCredit}
+        />
+      </td>
+    ) : null}
+    {noGrouping ? (
+      <td>
+        {releaseGroup.typeName
+          // TODO: replace with releaseGroupType when it works here
+          ? releaseGroup.l_type_name
+          : null
+        }
+      </td>
+    ) : null}
+    {noRatings ? null : (
+      <td className="c">
+        <RatingStars entity={releaseGroup} />
+      </td>
+    )}
+    <td className="c">{releaseGroup.release_count}</td>
+  </tr>
+);
+
+const ReleaseGroupsList = ({
+  $c,
+  checkboxes,
+  noGrouping,
+  noRatings,
+  order,
+  releaseGroups,
+  seriesItemNumbers,
+  showArtists,
+  sortable,
+}: ReleaseGroupsListProps) => {
+  const groupedReleaseGroups = groupBy(releaseGroups, 'typeName');
+  return (
+    noGrouping ? (
+      <table className="tbl release-group-list">
+        <ReleaseGroupsListHeader
+          $c={$c}
+          checkboxes={checkboxes}
+          noGrouping={noGrouping}
+          noRatings={noRatings}
+          order={order}
+          seriesItemNumbers={seriesItemNumbers}
+          showArtists={showArtists}
+          sortable={sortable}
+        />
+        <tbody>
+          {releaseGroups.map((releaseGroup, index) => (
+            <ReleaseGroupsListEntry
+              $c={$c}
+              checkboxes={checkboxes}
+              index={index}
+              key={releaseGroup.id}
+              noGrouping={noGrouping}
+              noRatings={noRatings}
+              releaseGroup={releaseGroup}
+              seriesItemNumbers={seriesItemNumbers}
+              showArtists={showArtists}
+            />
+          ))}
+        </tbody>
+      </table>
+    ) : (
+
+      Object.keys(groupedReleaseGroups).map((type) => {
+        const releaseGroupsOfType = groupedReleaseGroups[type];
+        return (
+          <React.Fragment key={type}>
+            <h3>
+              {type === 'null'
+                ? l('Unspecified type')
+                // TODO: replace with releaseGroupType when it works here
+                : releaseGroupsOfType[0].l_type_name
```

Comment:
Same as line 160, plus a bit of a hack to get the actual translated type (the releaseGroupType alternative would still use releaseGroupsOfType[0], since that function takes an RG, not a type string).